.github/workflows/DeployActions.yml

```
@@ -0,0 +1,41 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+# branches: [ $default-branch, Testing/GithubActions, feature/homework-2 ]
+name: Deploy Actions
+
+on:
+  push:
+  pull_request:
+
+defaults:
+    run:
+      working-directory: ./homeworks/homework-vue
+
+jobs:
+  lint:
+    name: Run lint
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v1
+    - run: npm ci
+    - run: npm run lint --fix
```

```suggestion
    - run: npm run lint
```
Or do you want your CI server to contribute to your code? :stuck_out_tongue: